client/ayon_houdini/plugins/publish/collect_arnold_rop.py

```diff
@@ -160,3 +161,12 @@ def replace(match):
                 os.path.join(dir, (file % i)).replace("\\", "/"))
 
         return expected_files
+
+    def evalParmNoFrame(self, rop, parm, **kwargs):
+        try:
+            return evalParmNoFrame(rop, parm, **kwargs)
+        except Exception as exc:
```

Comment:
I vaguely recall that `hou.Error` does not inherit from `Exception` - is that still the case? If so, we may need to capture `hou.Error` explicitly if the library function does not?

Reply:
`evalParmNoFrame`  function has `assert` and `RuntimeError`  and they'll be raised before the `PublishError`.
I don't know if that is sufficient.